common/response.go

```diff
@@ -133,15 +135,20 @@ func NewHTTPResponse(ctx context.Context, req *Request, resp *network.Response,
 }
 
 func (r *Response) fetchBody() error {
+	r.bodyMx.RLock()
 	if r.body != nil || r.request.frame == nil {
+		r.bodyMx.RUnlock()
 		return nil
 	}
+	r.bodyMx.RUnlock()
 	action := network.GetResponseBody(r.request.requestID)
 	body, err := action.Do(cdp.WithExecutor(r.ctx, r.request.frame.manager.session))
 	if err != nil {
 		return err
```

Comment:
Let's enhance the error message while in here.

Reply:
Done in 8718977.